Makefile

```diff
@@ -59,6 +59,13 @@ TEST_PRIVATE_GITLAB ?=
 # private repository authentication secret
 TEST_SOURCE_SECRET ?=
 
+# Image settings for building and pushing images
+IMAGE_HOST ?= quay.io
+IMAGE ?= shipwright/shipwright-operator
```

Comment:
I am a little late. :-) Is `shipwright-operator` the right choice or would `build-operator` be better? What if in the future we have more than one operator ?

Reply:
If that comes to be, I recommend a naming scheme that prefixes "shipwright" and describes the capability being delivered via the operator. Ex: `shipwright-build-operator`.

Reply:
+1, at least, the `shipwright` is the org name, not the project/service name.